DOCS.md

```diff
@@ -359,14 +360,15 @@ __Arguments__
 ---------------------------------------
 
 <a name="getThreadList" />
-### api.getThreadList(start, end, callback)
+### api.getThreadList(start, end, isPending, callback)
 
 Will return information about threads.
 
 __Arguments__
 
 * `start`: Start index in the list of recently used threads.
 * `end`: End index.
+* `isPending`: Boolean indicating that it will return the thread list of message requests.
```

Comment:
Can you phrase this a little more clearly?
I think "Optional boolean, set to true if you want the list of message requests." conveys the behaviour more explicitly.
I'm also wondering if we should just call it like.. "requests" or something?  I'm not totally sure that pending is obviously associated with message requests.  Shouldn't matter too much with the explanation alongside though, so I don't really care.